lightningd/dual_open_control.c

```diff
@@ -3517,6 +3521,10 @@ static void handle_commit_received(struct subd *dualopend,
 		was_pending(command_success(cmd, response));
 		return;
 	case REMOTE:
+		if (!updated) {
+			log_info(channel->log, "Already had sigs, skipping notif");
+			return;
+		}
```

Comment:
!!Thanks!! :orange_heart:  I saw this intermittantly and couldn't figure out what was happening!